logdna/logdna.go

```diff
@@ -64,59 +92,33 @@ func init() {
 
 // NewLogDNAAdapter creates adapter:
 func NewLogDNAAdapter(route *router.Route) (router.LogAdapter, error) {
-    token := os.Getenv("LOGDNA_KEY")
-    if token == "" {
+    logdnaKey := os.Getenv("LOGDNA_KEY")
+    if logdnaKey == "" {
         return nil, errors.New("Cannot Find Environment Variable \"LOGDNA_KEY\"")
     }
 
-    config := adapter.Configuration{
-        Custom:         adapter.CustomConfiguration{
-            Endpoint:   getStringOpt("LOGDNA_URL", "logs.logdna.com/logs/ingest"),
-            Hostname:   getStringOpt("HOSTNAME", ""),
-            Tags:       getStringOpt("TAGS", ""),
-            Token:      token,
-            Verbose:    os.Getenv("VERBOSE") != "0",
-        }, HTTPClient:  adapter.HTTPClientConfiguration{
-            DialContextKeepAlive:   getDurationOpt("DIAL_KEEP_ALIVE", 60) * time.Second,        // 30 by Default
-            DialContextTimeout:     getDurationOpt("DIAL_TIMEOUT", 60) * time.Second,           // 30 by Default
-            IdleConnTimeout:        getDurationOpt("IDLE_CONN_TIMEOUT", 60) * time.Second,      // 90 by Default
-            Timeout:                getDurationOpt("HTTP_CLIENT_TIMEOUT", 60) * time.Second,    // 30 by Default
-            TLSHandshakeTimeout:    getDurationOpt("TLS_HANDSHAKE_TIMEOUT", 30) * time.Second,   // 10 by Default
-        }, Limits:      adapter.LimitConfiguration{
-            FlushInterval:      getDurationOpt("FLUSH_INTERVAL", 250) * time.Millisecond,
-            MaxBufferSize:      getUintOpt("MAX_BUFFER_SIZE", 2) * 1024 * 1024,
-            MaxLineLength:      getUintOpt("MAX_LINE_LENGTH", 16000),
-            MaxRequestRetry:    getUintOpt("MAX_REQUEST_RETRY", 10),
-        },
+    hostname := os.Getenv("HOSTNAME")
+    if hostname == "" {
+        host, _ := os.Hostname()
```

Comment:
I know it's a rare event but perhaps we should still check that the hostname call was successful?

Reply:
Done!